### PR TITLE
Improve content types titles in some templates

### PR DESCRIPTION
**Pull request by lucasmezencio**
It is a simple PR.
It is just to improve Content Types readability on the Administration Panel on some templates.

### CODE REVIEW

File: app/view/twig/_sub/_record_list.twig [hunk 1]
```diff
@@ -57,7 +57,7 @@
                     {% set lastgroup = content.group.name %}
                 {% endif %}
             {% else %}
-                {{ __('contenttypes.generic.none-available', {'%contenttypes%': contenttype.slug}) }}
+                {{ __('contenttypes.generic.none-available', {'%contenttypes%': contenttype.singular_name}) }}
```
Comment (GawainLynch) [inline app/view/twig/_sub/_record_list.twig]: This one should be `contenttype.name`

File: app/view/twig/components/panel-lastmodified.twig [hunk 2]
```diff
@@ -13,7 +13,7 @@
     {% if context.filtered|default %}
         {{ 'Recent modifications' }}
     {% else %}
-        {{ __('contenttypes.generic.last-modified', {'%contenttypes%': context.contenttype.slug}) }}
+        {{ __('contenttypes.generic.last-modified', {'%contenttypes%': context.contenttype.singular_name}) }}
```
Comment (GawainLynch) [inline app/view/twig/components/panel-lastmodified.twig]: This one should be `contenttype.name`

File: app/view/twig/components/panel-lastmodified.twig [hunk 3]
```diff
@@ -45,7 +45,7 @@
                     <small><a href="{{ path('changelogrecordsingle', {'contenttype': context.contenttype.slug, 'contentid': entry.contentid, 'id': entry.id}) }}">view</a></small>
                 </li>
             {% else %}
-                <li><em>{{ __('contenttypes.generic.no-latest', {'%contenttype%': context.contenttype.slug}) }}.</em></li>
+                <li><em>{{ __('contenttypes.generic.no-latest', {'%contenttype%': context.contenttype.singular_name}) }}.</em></li>
```
Comment (GawainLynch) [inline app/view/twig/components/panel-lastmodified.twig]: This one should be `contenttype.name`

File: app/view/twig/components/panel-lastmodified.twig [hunk 4]
```diff
@@ -55,7 +55,7 @@
                     <small>({{ buic_moment(content.datechanged) }})</small>
                 </li>
             {% else %}
-                <li><em>{{ __('contenttypes.generic.no-latest', {'%contenttype%': context.contenttype.slug}) }}.</em></li>
+                <li><em>{{ __('contenttypes.generic.no-latest', {'%contenttype%': context.contenttype.singular_name}) }}.</em></li>
```
Comment (GawainLynch) [inline app/view/twig/components/panel-lastmodified.twig]: This one should be `contenttype.name`

File: app/view/twig/overview/_panel-actions_overview.twig [hunk 5]
```diff
@@ -7,7 +7,7 @@
 
 {% block panel_icon 'fa-cog' %}
 
-{% block panel_head __('contenttypes.generic.actions-all', {'%contenttypes%': context.contenttype.slug}) %}
+{% block panel_head __('contenttypes.generic.actions-all', {'%contenttypes%': context.contenttype.singular_name}) %}
```
Comment (GawainLynch) [inline app/view/twig/overview/_panel-actions_overview.twig]: This one should be `contenttype.name`

**Comment by GawainLynch** [comment]
Welcome @lucasmezencio and thank you! :+1: 

There are a few of these that are definitely referring to the plural, so I'll add them as "review comments" in a minute.

**Comment by lucasmezencio** [comment]
Good, @GawainLynch! Done! 😃

**Comment by GawainLynch** [comment]
Legendary, thank you again! :+1:

**Pull request merged**